app/models/form_submission_attempt.rb

```diff
@@ -25,14 +25,21 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        log_info = { form_submission_id:,
+                     benefits_intake_uuid: form_submission&.benefits_intake_uuid,
+                     form_type: form_submission&.form_type }
```

Comment:
Thanks for pulling this out. I like it!

Reply:
only pulled it out because needed to use it in multiple places after putting in my additions, since you only were using it once, it made sense how you had it for yours, all good!